### PR TITLE
[feat]: Added Static Landing Page

### PR DESCRIPTION
**Pull request by raghav2110**
Hi , I have added static landing page for ecom website. Please review it. Cant provide review for now.

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+export function Navbar(){
+    return(
+        <header className="header-main">
+        <nav className="nav nav-dark" style={{position: "relative"}}>
```
Comment (omjadhav85) [inline src/components/Navbar.jsx]: would suggest to add the inline style in the class as well

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+//  import { Link } from "react-router-dom";
+
+
```
Comment (saadahmadhere) [inline src/components/Footer.jsx]: remove extra space from here

File: src/pages/Home/HomePage.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                        <div className="grid-category">
+                            <img src="https://res.cloudinary.com/dxsodik0q/image/upload/v1648042630/Sneaker%20Street/new-arrival-1_fxnt2k.jpg" className="img-responsive" alt="" />
+                            <div className="product-detail">
+                                <div className="brand-name">Nike</div>
+                                <div className="product-name">NMD_R1 - CORE BLACK</div>
+                                <div className="price-tag">₹12,999</div>
+                            </div>
+                        </div>
+                    </a>
+                    <a>
```
Comment (omjadhav85) [inline src/pages/Home/HomePage.jsx:29]: Better to divide Hero part and category sections into different components and then only added those components here. Will look much cleaner.

**Review by saadahmadhere — APPROVED** [review]
Clean & sleek. Well organized code, variables say it all.

**Review by omjadhav85 — APPROVED** [review]
Suggested some minor refactoring changes. Else looks good to merge...great work!!!

File: src/components/Navbar.jsx (hunk 1 above)
Comment (raghav2110) [reply src/components/Navbar.jsx]: Okay will do that

File: src/components/Footer.jsx (hunk 2 above)
Comment (raghav2110) [reply src/components/Footer.jsx]: Noted.

File: src/pages/Home/HomePage.jsx (hunk 3 above)
Comment (raghav2110) [reply src/pages/Home/HomePage.jsx:29]: Can try this. Thanks for suggestion

**Pull request merged**